docs/content/en/release-notes/_index.md

```diff
@@ -38,15 +38,15 @@ Release date: 27-August-2021
 The K8ssandra helm chart deploys the following components. Some are optional, and depending on the configuration, may not be deployed:
 
 * [Apache Cassandra](https://cassandra.apache.org/) - the deployed version depends on the configured `cassandra.version` setting:
-  * 4.0.0 (default)
+  * 4.0.1 (default)
   * 3.11.11
   * 3.11.10
   * 3.11.9
   * 3.11.8
   * 3.11.7
-* DataStax Kubernetes Operator for Apache Cassandra ([cass-operator](https://github.com/k8ssandra/cass-operator)) 1.7.1
+* [cass-operator](https://github.com/k8ssandra/cass-operator)) 1.8.0.
```

Comment:
This should go to 1.9.0 before we do the k8ssandra release.

Reply:
Updated to anticipate 1.9.0 - thanks.